src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/ArgumentSuggestions.cs

```diff
@@ -40,8 +42,13 @@ internal static class ArgumentSuggestions
                 { typeof(ValueFunction), LanguageCodeSuggestion }
             }, isThreadSafe: true);
 
-        public static IEnumerable<KeyValuePair<string, DType>> GetArgumentSuggestions(TryGetEnumSymbol tryGetEnumSymbol, bool suggestUnqualifiedEnums, TexlFunction function, DType scopeType, int argumentIndex, out bool requiresSuggestionEscaping)
+        public static IEnumerable<KeyValuePair<string, DType>> GetArgumentSuggestions(IntellisenseData.IntellisenseData intellisenseData, TryGetEnumSymbol tryGetEnumSymbol, bool suggestUnqualifiedEnums, TexlFunction function, DType scopeType, int argumentIndex, out bool requiresSuggestionEscaping)
```

Comment:
>GetArgumentSuggestions [](http://example.com/codeflow?start=63&length=22)

Please make sure that the places that call this function from the Power Apps repository are updated before checking this in.